src/Data/Parameterized/WidthRepr.hs

```diff
@@ -0,0 +1,35 @@
+module Data.Parameterized.WidthRepr
```

Comment:
This module / type could use documentation.  It is a `NatRepr` plus mask, right?

Reply:
I'm not even sure we want this module at all.

Reply:
It's an interesting idea, but let's remove it if we aren't using it.  If we decide we want this, it should go in parameterized-utils anyway.